shell/platform/common/accessibility_bridge.cc

```diff
@@ -43,7 +43,29 @@ void AccessibilityBridge::AddFlutterSemanticsCustomActionUpdate(
 }
 
 void AccessibilityBridge::CommitUpdates() {
+  // AXTree cannot move a node in a single update.
+  // This must be split across two updates:
+  //
+  // * Update 1: remove nodes from their old parents.
+  // * Update 2: re-add nodes (including their children) to their new parents.
+  //
+  // First, start by removing nodes if necessary.
+  std::optional<ui::AXTreeUpdate> remove_reparented =
+      CreateRemoveReparentedNodesUpdate();
+  if (remove_reparented.has_value()) {
+    tree_.Unserialize(remove_reparented.value());
+
+    std::string error = tree_.error();
+    if (!error.empty()) {
+      BASE_LOG() << "Failed to update ui::AXTree, error: " << error;
```

Comment:
Prefer `FML_LOG(ERROR)` here.

Reply:
Fixed this plus the other log in this file